README.md

```
@@ -269,7 +278,7 @@ The application uses the following main models:
 
 ## Project Structure
 
-\`\`\`
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)